learn/configuration/displayed_searchable_attributes.md

```diff
@@ -25,8 +25,12 @@ Meilisearch uses an ordered list to determine which attributes are searchable. T
 
 In other words, the `searchableAttributes` list serves two purposes:
 
-1. It designates the fields that are searchable.
-2. It dictates the [attribute ranking order](/learn/core_concepts/relevancy.md#attribute-ranking-order).
+1. It designates the fields that are searchable
+2. It dictates the [attribute ranking order](/learn/core_concepts/relevancy.md#attribute-ranking-order)
+
+::: warning
```

Comment:
I would vote for removing this warning. Although this behavior involves `searchableAttributes`, it only affects people using attributes to highlight and matches. Therefore, we only need to give a warning in pages/sections related to those parameters.

Reply:
That makes sense